torch/_dynamo/config.py

```diff
@@ -516,12 +513,12 @@ def default_debug_dir_root():
     "storage_offset",
     "layout",
     "data",
+    "is_neg",
+    "is_conj",
+    "is_pinned",
+    "is_contiguous",
```

Comment:
Let's try to do the following in some follow ups:
- we can delete .data (I don't know why that's here)
- we can delete requires_grad (long story, but AOTAutograd doesn't support create_graph=True, so the gradients never have requires_grad=True. HOWEVER we should check to see the "example" gradients that we're passing actually have requires_grad=False (if they don't then they need an extra .detach() call)